client/src/components/CommentApp/main.scss

```diff
@@ -1,122 +1,10 @@
-@use '../../../scss/settings/variables' as *;
-
-$color-comment-separator: theme('colors.border-furniture');
-
-$color-box-background: theme('colors.surface-page');
-$color-box-background-focused: theme('colors.surface-page');
-$color-box-border: theme('colors.border-furniture');
-$color-box-border-focused: theme('colors.border-field-hover');
-$color-box-text: theme('colors.text-context');
-$box-border-radius: 5px;
-$box-padding: 20px;
-
-@mixin focus-outline {
-  outline: theme('colors.focus') solid 3px;
-}
-
-@mixin box {
```

Comment:
Unless otherwise noted, all the code here has been kept, just moved directly inside the `Comment` component styles.